app/assets/stylesheets/components/firm/_all.scss

```diff
@@ -1,2 +1,3 @@
 @import 'settings';
 @import 'firm';
+@import 'firm_list';
```

Comment:
Yeah exactly, the directory contains the component & any settings. So unless `firm_list` was a child of `firm` (which it isn't) then it deserves its own directory.